configs/hoodi/lidov3-testnet.yaml

```diff
@@ -0,0 +1,1994 @@
+        STETH: *lido
+        WSTETH: *wstETH
+        balanceOf: null
+        bunkerModeSinceTimestamp: 0
+        calculateFinalizationBatches: null
+        findCheckpointHints: null
+        getApproved: null
+        getBaseURI: ""
+        getClaimableEther: null
+        getContractVersion: *MAX_UINT256
+        getLastCheckpointIndex: 0
+        getLastFinalizedRequestId: 0
+        getLastRequestId: 0
+        getLockedEtherAmount: 0
+        getNFTDescriptorAddress: *ZERO_ADDRESS
+        getResumeSinceTimestamp: 0
+        getRoleAdmin: null
+        getRoleMember: null
+        getRoleMemberCount: null
+        getWithdrawalRequests: null
+        getWithdrawalStatus: null
+        hasRole: null
+        isApprovedForAll: null
+        isBunkerModeActive: true
+        isPaused: false
+        name: "Lido: stETH Withdrawal NFT"
+        ownerOf: null
+        prefinalize: null
+        supportsInterface: null
+        symbol: "unstETH"
+        tokenURI: null
+        unfinalizedRequestNumber: 0
+        unfinalizedStETH: 0
+
+    withdrawalVault:
+      name: WithdrawalVault
+      address: *withdrawalVault
+      proxyName: WithdrawalsManagerProxy
+      implementation: *withdrawalVaultImplAddress
+      proxyChecks:
+        implementation: *withdrawalVaultImplAddress
+        proxy_getAdmin: "0xd401Bf29751aE77cF5A479b22fbAaB30cD027dD6"
+        proxy_getIsOssified: false
+      checks:
+        LIDO: *lido
+        TREASURY: *aragonAgent
+        getContractVersion: 0
+      implementationChecks:
+        LIDO: *lido
+        TREASURY: *aragonAgent
+        getContractVersion: *MAX_UINT256
+
+    burner:
+      name: Burner
+      address: *burner
+      checks:
+        DEFAULT_ADMIN_ROLE: *ZERO_BYTES32
+        LIDO: *lido
+        LOCATOR: *lidoLocator
+        REQUEST_BURN_MY_STETH_ROLE: *REQUEST_BURN_MY_STETH_ROLE
+        REQUEST_BURN_SHARES_ROLE: *REQUEST_BURN_SHARES_ROLE
+        getCoverSharesBurnt: null
+        getExcessStETH: null
+        getNonCoverSharesBurnt: null
+        getRoleAdmin: null
+        getRoleMember: null
+        getRoleMemberCount: null
+        getSharesRequestedToBurn: null
+        hasRole: null
+        supportsInterface: null
+
+    minFirstAllocationStrategy:
+      name: MinFirstAllocationStrategy
+      address: *minFirstAllocationStrategy
+      checks:
+        allocate: null
+
+    accounting:
+      name: Accounting
+      address: *accounting
+      proxyName: OssifiableProxy
+      implementation: *accountingImplAddress
+      proxyChecks:
+        proxy__getAdmin: *aragonAgent
+        proxy__getImplementation: *accountingImplAddress
+        proxy__getIsOssified: false
+      checks:
+        LIDO: *lido
+        LIDO_LOCATOR: *lidoLocator
+        simulateOracleReport: null
+        handleOracleReport: null
+      implementationChecks:
+        LIDO: *lido
+        LIDO_LOCATOR: *lidoLocator
+        simulateOracleReport: null
+        handleOracleReport: null
+
+    # vaultHub:
```

Comment:
:question:

Reply:
This is github issue, for some reason it uses local `hoodi` branch instead of `Alexvozhak:hoodi`...